src/Pages/Login/Login.js

```diff
@@ -1,12 +1,51 @@
-import React, { Component } from "react";
+import React from "react";
+import "./Login.scss";
+import "../../Styles/reset.scss";
 
-class Login extends Component {
+class Login extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      checkbox: false,
+    };
   }
+  IsChecked = () => {
```

Comment:
함수명은 camelCase형태로 작성합니다. 
고로 , 맨 첫글자는 아래와 같이 소문자로 시작해주세요 :) 
```suggestion
  isChecked = () => {
```

Reply:
넵 수정하겠습니다